src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_backup_commands.py

```diff
@@ -427,6 +427,7 @@ def test_backup_protection(self, resource_group, vault_name, vm_name):
         protection_check = self.cmd('backup protection check-vm --vm-id {vm_id}').output
         self.assertTrue(protection_check == '')
 
+    @unittest.skip('more than 3 hours')
```

Comment:
With the skip, CI also will skip the test, right? In playback mode, does it also take more than 3 hours?